PlayerDeck.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Card_Game
+{
+    public class PlayerDeck : Deck
+    {
+        public Card GetTopCard()
+        {
+            return Cards.Last();
+        }
+        public void RemoveTopCard()
```

Comment:
Same here. How about moving it to Deck abstract class? I think it may be used for both PlayerDeck and TableDeck.